src/Components/SearchBar.jsx

```diff
@@ -44,11 +47,14 @@ export default function SearchBar({onClickMenuCb, showNavPanel}) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [searchParams])
 
-
   const onSubmit = (event) => {
     // Prevent form event bubbling and causing page reload.
     event.preventDefault()
-
+    if (isValidModelURL(inputText)) {
```

Comment:
Per discussion, I think we need to add more here for users who aren't quite getting it right.  I suggest one more check here:

```
if (looksLikeURL(inputText)) {
  if (isValidModelURL(inputText)) {
     ...
  } else {
     // help user
  }
```
and looksLikeURL can be something simple like inputText.startsWith('http://')